src/main/java/kostka/moviecatalog/factory/RuntimeConfigurationFactory.java

```diff
@@ -0,0 +1,16 @@
+package kostka.moviecatalog.factory;
+
+import kostka.moviecatalog.dto.RuntimeConfigDto;
+import kostka.moviecatalog.entity.runtimeconfiguration.VisibleMoviesConfigValue;
+
+public class RuntimeConfigurationFactory {
+
+    public static Object getValueForUpdateRuntimeConfig(final RuntimeConfigDto dto) {
```

Comment:
This is not ideal. This implementation requires the fields of all possible runtime configuration will be gather together in DTO. Probably you should have a separate DTO of each runtime configuration and create a runtime configuratino from dto of the same type. (You can have a method `VisibleMoviesConfigValue.from(dto)` directly in VisibleMoviewConfigValue). But it will require another refactoring probably. If you can, you can plan a meeting to discuss that. ;)